### PR TITLE
patient update

### PR DESCRIPTION
**Pull request by RunjeethNikam**

### CODE REVIEW

File: src/app/store.js [hunk 1]
```diff
@@ -6,15 +6,10 @@ const initialState = {};
...
-let args = [ applyMiddleware(...middleware),  ]
-if(process.env.NODE_ENV !== 'development'){
-    args.pop()
-}
-
 const store = createStore(
     rootReducer,
     initialState,
-    compose(...args)
+    compose(applyMiddleware(...middleware), window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
```
Comment (gauravksingh) [inline src/app/store.js:12]: DEV tools should only be used in dev, so use environment variable here.

File: src/components/Inputs/MultiSelectDateDropdown/index.js [hunk 2]
```diff
@@ -23,11 +24,17 @@ export default function MultiSelectDateDropdown({ onSelect, fieldName }) {
     if (e.which === 13) {
       // Enter key pressed
       let value = `${fieldName}: ${subDropdownValue} ${fromValue}`;
+      let value_return = {
```
Comment (gauravksingh) [inline src/components/Inputs/MultiSelectDateDropdown/index.js:27]: Please think of better variable name.

File: src/screens/Patients.js [hunk 3]
```diff
@@ -4,9 +4,16 @@ import { Grid } from '@material-ui/core';
 import Header from 'Containers/Header';
 import PatientsList from 'Containers/PatientsList';
 import { Search } from 'Components/Inputs';
+import Button from "@material-ui/core/Button";
+import {ListAlt} from "@material-ui/icons";
```
Comment (gauravksingh) [inline src/screens/Patients.js:8]: Formatting. Spaces are missed at few other places as well,

File: src/containers/PatientsList/index.js [hunk 4]
```diff
@@ -94,14 +94,13 @@ export function PatientsList( props ) {
 
       update_patients.forEach((attr) => {
         let date = new Date(attr[CLINICAL_STATUS_UPDATED_AT])
-        attr[CLINICAL_STATUS_UPDATED_AT] = moment(date).fromNow();
+        attr[CLINICAL_STATUS_UPDATED_AT] = moment(date).format('DD-MM-YYYY h:mm a');
```
Comment (gauravksingh) [inline src/containers/PatientsList/index.js:97]: use constant for date format. I believe Ayush Already created one.

File: src/utils/listFilter.js [hunk 5]
```diff
@@ -28,16 +28,16 @@ export const mapProps = (params, requiredLists) => {
...
 ) => {
-    let updateSelectedParams = {
+    const updateSelectedParams = {
         ...selectedParams,
-        ...val,
+        ...mapProps(val, requiredLists),
     };
-
-     return mapProps(updateSelectedParams, requiredLists);
+    console.log(val, {...updateSelectedParams})
```
Comment (gauravksingh) [inline src/utils/listFilter.js:39]: remove this

File: src/containers/NavigationPanel/index.js [hunk 6]
```diff
@@ -61,33 +61,33 @@ export function NavigationPanel(props) {
...
-                </Link>
-              </li>
-              <li className={getActivatedRoute('/patients/private-quarantine') ? 'active' : ''}>
-                <Link to={'/patients/private-quarantine'} className="nav-link">
-                  {i18n.t('Private Quarantine')}
-                </Link>
-              </li>
-            </ul>
+            {/*<ul className="navbar-nav sub-nav clearfix">*/}
+              {/*<li className={getActivatedRoute('/patients/add') ? 'active' : ''}>*/}
```
Comment (shashankJTG) [inline src/containers/NavigationPanel/index.js:65]: Why is this commented? Please uncomment the same

File: src/utils/listFilter.js (hunk 5 above)
Comment (shashankJTG) [inline src/utils/listFilter.js:39]: remove console

File: src/containers/NavigationPanel/index.js (hunk 6 above)
Comment (RunjeethNikam) [reply src/containers/NavigationPanel/index.js:65]: @mintusah25 sir told to comment these line for the time being.

**Pull request merged**